### ISSUE TITLE
AWS service throws unauthorizes causing issues in RHPDS

### ISSUE THREAD

**Issue by NonyNo3lle** [ansible/workshops#182]
##### SUMMARY
This is not a 100% occurrence, but now and again we see the RHUI service in the EUROPE region throw an unauthorized error message, which causes deployments in RHPDS to fail time and again.
We usually recommend users try another region, but it is our understanding you can work this with AWS support, you need to open a ticket in your account and follow their guidance.

We must verify, at least, that configuration for these servers is equal to their counterparts in APAC and NA:

rhui2-cds01.eu-central-1.aws.ce.redhat.com/
rhui2-cds02.eu-central-1.aws.ce.redhat.com/

The error in AAD logs is: 

REGION-jbcs-rhui-rhel-7-rpms: [Errno 256] No more mirrors to try.\nhttps://rhui2-cds01.eu-central-1.aws.ce.redhat.com/pulp/repos//content/dist/rhel/rhui/server/7/7Server/x86_64/jbcs/1/os/repodata/repomd.xml: [Errno 14] HTTPS Error 401 - Unauthorized\nhttps://rhui2-cds02.eu-central-1.aws.ce.redhat.com/pulp/repos//content/dist/rhel/rhui/server/7/7Server/x86_64/jbcs/1/os/repodata/repomd.xml: [Errno 14] HTTPS Error 401 - Unauthorized"}
c696-student1-ansible      : ok=9    changed=6    unreachable=0    failed=1

##### ISSUE TYPE
 - Bug Report

##### EXTRA VARS FILE
N/A

##### ANSIBLE VERSION
```2.6.4
```

**Comment by IPvSean** [comment]
issue opened with AWS, waiting on response

unsuccessful on trying to replicate, will continue testing on eu-central-1 for all development going forward until we figure out what is going on.

**Comment by IPvSean** [comment]
updated aws ticket with additional info, still waiting response from aws.  the rhui servers are controlled by another group at red hat, they have to open another ticket which will make the tickets go full circle :)

**Comment by IPvSean** [comment]
found out contact internally is Jerome Boutaud

**Comment by IPvSean** [comment]
@NonyNo3lle is this still an issue?

**Comment by NonyNo3lle** [comment]
@IPvSean Allow me a couple days to check on status with Ops, sorry about the delay

**Comment by IPvSean** [comment]
no response from submitter for 22 days
